README.rst

```diff
@@ -0,0 +1,105 @@
+image here
+image target here
+
+image here
+image target here
+
+
+PDFmorph
+========================================================================
+add description, sample below
+Code package for morphin' atomic pair distribution functions.
+
+PDFmorph is a user-friendly software package. [more info here]
+
+
+REQUIREMENTS
+------------------------------------------------------------------------
+
+PDFmorph requires Python 3.7 and several third party libraries 
+that are used by PDFmorph and its components.
+
+* setuptools         - tools for installing and setting up Python packages
+* Sphinx             - documentation generator for reStructuredText files
+* sphinx_rtd_theme   - user-friendly sphinx theme
+* Doctr              - deploys docs from Travis CI to GitHub pages
+* pip                - package installer for Python
+* NumPy              - library for scientific computing with Python
+* matplotlib         - Python 2D plotting library
+* SciPy              - library for highly technical Python computing
+* diffpy.utils       - shared helper utilities for wx GUI, 
+https://github.com/diffpy/diffpy.utils
+* Flake8             - tool for performing static analysis of source code to
```

Comment:
I doubt most of these below are requirements.  Also pip, doctr, etc..  This list needs a lot of cleaning.

Reply:
This is another one I asked @CJ-Wright about and put together after referencing the requirements directory in the GitHub

Reply:
please resolve